app/views/_flash_msg.html.erb

```diff
@@ -7,3 +7,14 @@
     <% flash.delete(type) %>
   <% end %>
 <% end %>
+
+<% if form_errors? %>
```

Comment:
Having the helper include the `@form` and then assuming it on line 14 below seems like we might have knowledge spread across too many places.

What if we had `<% if @form&.errors&.any? %>` here?  That would consolidate that logic into the view.

Reply:
And based on that, I'd begin to wonder about the helper methods that reference `@form`

Reply:
I like it!